android/pytorch_android/src/main/cpp/pytorch_jni_common.cpp

```diff
@@ -8,8 +8,7 @@
 
 #include "pytorch_jni_common.h"
 #if defined(__ANDROID__)
-#include <caffe2/utils/threadpool/ThreadPool.h>
-#include <caffe2/utils/threadpool/ThreadPoolMobile.h>
+#include <caffe2/utils/threadpool/pthreadpool-cpp.h>
```

Comment:
This new threadpool is not mobile specific since we allow NNPACK, QNNPACK, and XNNPACK on non-mobile builds.  In reality this thread pool is just a simple wrapper around pthreadpool so there is nothing mobile specific about it.  I am open to better names if you have any suggestions.